content/blog/2024-05-28-announcing-810-stable.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "General Availability of AlmaLinux 8.10 Stable!"
+type: blog
+author:
+ name: "Eduard Abdullin"
+ bio: "The Release and Automation Engineer."
```

Comment:
This sounds odd. It should be adjusted:

> Release and Automation Engineer

If you want to communicate you're the lead, perhaps `Principal Release and Automation Engineer` or `Lead Release and Automation Engineer` would be better.

Reply:
Fixed